test/parallel_api/algorithm/alg.merge/merge.pass.cpp

```diff
@@ -165,5 +175,26 @@ main()
     test_algo_basic_double<std::int32_t>(run_for_rnd_fw<test_non_const<std::int32_t>>());
 #endif
 
+    using T = std::tuple<std::int32_t, std::int32_t>; //a pair (key, value)
+    std::vector<T> a = { {1, 2}, {1, 2}, {1,2}, {1,2}, {1, 2}, {1, 2} };
+    std::vector<T> b = { {1, 1}, {1, 1}, {1, 1}, {1, 1}, {1, 1}, {1, 1}, {1, 1} };
+    std::vector<T> merged(a.size() + b.size());
+
+    auto comp = [](auto a, auto b) { return std::get<0>(b) < std::get<0>(a); }; //greater by key
+
+    invoke_on_all_policies<6>()(test_merge_compare<T>(), a.begin(), a.end(), b.cbegin(), b.cend(), merged.begin(), comp,
```

Comment:
Could you wrap the case into some function, e.g. `test_merge_zip` and pass e.g. `100` as a kernel name augmentation in order to avoid unintentionally making kernel duplicates upon changing other parts of the test?

Reply:
rather `test_merge_tuple`, because we have not explicit zip_iterator usage here.